environment/k8s_environment.go

```diff
@@ -169,6 +199,38 @@ func (env *k8sEnvironment) TearDown() {
 	log.Error().Err(err)
 }
 
+// Writes logs for each container in a pod
+func writeLogsForPod(podsClient v1.PodInterface, pod coreV1.Pod, podFolder string) error {
+	for _, container := range pod.Spec.Containers {
+		logFile, err := os.Create(filepath.Join(podFolder, container.Name) + ".log")
+		if err != nil {
+			log.Err(err).Str("File Name", logFile.Name()).Msg("Error creating log file")
+		}
+
+		_, _ = logFile.WriteString("======================================================================================\n")
+		_, _ = logFile.WriteString("Test file name: " + ginkgo.CurrentGinkgoTestDescription().FileName + "\n")
```

Comment:
The point of the log buffer was to remove the need for importing Ginkgo in the environment, as this package should not be tied to the implementation of tests.

Reply:
Forgot to rip this out, already have this info in our unit reports